integration_tests/integration/bookedtoday/unexpectedArrivals/searchForExistingRecord.spec.ts

```diff
@@ -0,0 +1,77 @@
+import Page from '../../../pages/page'
+import Role from '../../../../server/authentication/role'
+import SearchForExistingPage from '../../../pages/bookedtoday/unexpectedArrivals/searchForExistingRecord'
+import NoRecordFoundPage from '../../../pages/bookedtoday/unexpectedArrivals/noRecordFound'
+import RecordFoundPage from '../../../pages/bookedtoday/unexpectedArrivals/recordFound'
+import PossibleRecordsFoundPage from '../../../pages/bookedtoday/unexpectedArrivals/possibleRecordsFound'
+
```

Comment:
An annoying comment but maybe `noRecordFound`, `recordFound` and `possibleRecordFound` naming could be changed to `noRecordsFound`, `singleRecordFound` and `multipleRecordsFound` to be consistent with other file names - in the next PR?

Reply:
could be, but i wanted to keep the same as the name of endpoints which are no-record-found, record-found and possible-records-found.